### PR TITLE
 Add example workflow for PA:USER_FILE variable 

### PR DESCRIPTION
**Pull request by tao-xinxiu**
The task transfers and shows the selected file content in the job result

### CODE REVIEW

File: GetStarted/resources/catalog/File_Upload.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.12" xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"  name="File_Upload" projectName="1. Basic Workflows" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
+  <variables>
+    <variable name="INPUT_FILE" value="" model="PA:USER_FILE"/>
+  </variables>
+  <description>
+    <![CDATA[ A workflow that shows the content of an user data space file. ]]>
```
Comment (aminelouati) [inline GetStarted/resources/catalog/File_Upload.xml:9]: an user --> a user

File: GetStarted/resources/catalog/File_Upload.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  </description>
+  <genericInformation>
+    <info name="workflow.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/file_trigger.png"/>
+  </genericInformation>
```
Comment (aminelouati) [inline GetStarted/resources/catalog/File_Upload.xml:13]: are you sure about the icon ?

File: GetStarted/resources/catalog/File_Upload.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  <taskFlow>
+    <task name="TransferAndDisplay" 
+    preciousResult="true" 
+    fork="true">
+      <description>
+        <![CDATA[ A Groovy task to transfer and display the content of an user data space file which is selected during job submission. ]]>
+      </description>
+      <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/file_trigger.png"/>
+      </genericInformation>
```
Comment (aminelouati) [inline GetStarted/resources/catalog/File_Upload.xml:23]: same as above

**Comment by aminelouati** [comment]
I think that you can improve this workflow example by showing how to use data management. It will demonstrate the utility of this model.

File: GetStarted/resources/catalog/File_Upload.xml (hunk 1 above)
Comment (fviale) [reply GetStarted/resources/catalog/File_Upload.xml:9]: A workflow that allows to upload a file and show its contents as job result.

**Comment by tao-xinxiu** [comment]
> I think that you can improve this workflow example by showing how to use data management. It will demonstrate the utility of this model.

The example uses already the data management by specifying the input file as variable.
```
<inputFiles>
    <files  includes="${INPUT_FILE}" accessMode="transferFromUserSpace"/>
</inputFiles>
```

Is it what you mean of showing how to use data management ?

File: GetStarted/resources/catalog/File_Upload.xml (hunk 2 above)
Comment (tao-xinxiu) [reply GetStarted/resources/catalog/File_Upload.xml:13]: Just checked with Denis. It's ok. Thanks.

**Comment by aminelouati** [comment]
You are right. Sorry

**Pull request merged**